packages/core/src/Stores/client-store.js

```diff
@@ -189,6 +190,7 @@ export default class ClientStore extends BaseStore {
             is_landing_company_loaded: observable,
             is_account_setting_loaded: observable,
             has_enabled_two_fa: observable,
+            has_changed_two_fa: observable,
```

Comment:
@shaheer-deriv Please update the types in `packages/stores/types.ts` 🙏🏻

Reply:
@farzin-deriv added 😄 👍